lib/node/NodeWatchFileSystem.js

```diff
@@ -58,9 +58,9 @@ class NodeWatchFileSystem {
 			removals = new Set(removals.filter(file => files.has(file)));
 			callback(
 				null,
-				changes.filter(file => files.has(file)).sort(),
-				changes.filter(file => dirs.has(file)).sort(),
-				changes.filter(file => missing.has(file)).sort(),
+				new Set(changes.filter(file => files.has(file))),
```

Comment:
That's an interface change (breaking change). You can't do that in the master branch. Add a TODO to change it to a Set in webpack 5.

Reply:
good point, I didn't catch that. I'll change this